docs/source/en-us/guide/quickstart.md

```diff
@@ -0,0 +1,360 @@
+title: Quick Start
+---
+
+# Quick Start
+
+This guide covers getting up and running with Egg using a real example.
+By following along with this guide step by step, you can quickly get started with Egg development.
+
+## Prerequisites
+
+- Operating System: Linux, OS X or Windows (not recommended).
+- Node.js Runtime: 6.x or newer; it is recommended that you use [LTS Releases][node].
+
+## the Quick Way
+
+To begin with, let's quickly initialize the project using a scaffold,
+which will quickly generate some of the major pieces of the application.
+
+```bash
+$ npm i egg-init -g
+$ egg-init egg-example --type=simple
+$ cd egg-example
+$ npm i
+```
+
+Then get up and running using the following commands.
+
+```bash
+$ npm run dev
+$ open localhost:7001
+```
+
+## Step by Step
+
+Usually you could just use [egg-init] of the last session,
+choose a scaffold that best fits your business model and quickly generate a project,
+then get started with the development.
+
+However, in this section, we will be manual instead and skip the scaffolds.
+We will build a project called [Egg HackerNews](https://github.com/eggjs/examples/tree/master/hackernews) step by step.
```

Comment:
However, in this section, instead of using scaffolds we will build a project called [Egg HackerNews](https://github.com/eggjs/examples/tree/master/hackernews) step by step, for a better understanding of how it works.